Evaluator/RealTime/instant_fluctuations_evaluator/instant_fluctuations.py

```diff
@@ -65,10 +65,14 @@ async def ohlcv_callback(self, exchange: str, exchange_id: str,
             self.average_volumes[segment] = np.mean(volume_data)
             self.average_prices[segment] = np.mean(price_data)
 
-        self.last_volume = volume_data[-1]
-        self.last_price = close_data[-1]
-        await self._trigger_evaluation(cryptocurrency, symbol,
-                                       evaluators_util.get_eval_time(full_candle=candle, time_frame=time_frame))
+        try:
+            self.last_volume = volume_data[-1]
+            self.last_price = close_data[-1]
+            await self._trigger_evaluation(cryptocurrency, symbol,
+                                           evaluators_util.get_eval_time(full_candle=candle, time_frame=time_frame))
+        except IndexError:
+            # candles data history is probably not yet available
```

Comment:
should we log in debug ?

Reply:
I don't think so, it happens so many times with websockets during the init phase XD
If there are really no candles, there will just be nothing happening at all in the bot, I think we can leave it like this

Reply:
It can be used to understand why instant fluctuation is not triggered after initialization